src/core/thread/mle.hpp

```diff
@@ -1767,6 +1781,13 @@ class Mle : public InstanceLocator, private NonCopyable
     void     SendAnnounce(uint8_t aChannel, bool aOrphanAnnounce, const Ip6::Address &aDestination);
 #if OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE
     otError SendLinkMetricsManagementResponse(const Ip6::Address &aDestination, LinkMetrics::LinkMetricsStatus aStatus);
+#endif
+#if OPENTHREAD_CONFIG_MAC_SSED_TO_SSED_LINK_ENABLE
+    otError SendLinkRequest(Child *aChild);
+    otError SendLinkAccept(const Ip6::MessageInfo &aMessageInfo,
+                           Child *                 aChild,
+                           const Challenge &       aChallenge,
+                           bool                    aRequest);
```

Comment:
Provide a default value `false` for `aRequest` for simpler method call and better readability?

Reply:
SGTM